labgrid/resource/udev.py

```diff
@@ -46,9 +46,8 @@ def poll(self):
                 break
             self.log.debug("%s: %s", device.action, device)
             for resource in self.resources:
-                self.log.debug(" %s", resource)
                 if resource.try_match(device):
-                    break
+                    self.log.debug(" matched successfully")
```

Comment:
same as above.

Reply:
Same as above, the log message extends the device log message a few lines above, on the same level as the message in `try_match()`.